requirements.txt

```diff
@@ -1,3 +1,5 @@
+openvino==2022.3.0
```

Comment:
It's better we move openvino install in runtime/openvino/README.md. The requirements.txt is for training.